src/main/java/piano/instruments/InstrumentsPanel.java

```diff
@@ -11,9 +11,19 @@ public InstrumentsPanel(MidiChannel midiChannel) {
         setLayout(new FlowLayout());
         setSize(MainFrameInterface.KEYBOARD_WIDTH, MainFrameInterface.INSTRUMENTS_PANEL_HEIGHT);
         setBackground(Color.BLACK);
-        for (InstrumentButton instrumentButton: new InstrumentButtonsArray()) {
-            instrumentButton.addActionListener(e -> midiChannel.programChange(instrumentButton.getInstrument()));
-            add(instrumentButton);
+        JComboBox combo = new JComboBox();
+
+
+        //instrument data from https://www.midi.org/specifications-old/item/gm-level-1-sound-set
+        String[] instrumentName = {"Acoustic Grand Piano", "Music Box", "Xylophone", "Accordion",
+                "Harmonica", "Acoustic Guitar", "Electric Guitar", "Violin", "Cello", "Trumpet",
+                "Flute"};
+        int[] instrumentVal = {1, 11, 14, 22, 23, 25, 27, 41, 43, 57, 74};
```

Comment:
These are known as "parallel arrays, and are typically a bad idea because if you have a mismatch between the two it's hard to figure it out.
When I tell you to sort them alphabetically, it gets more difficult than it should be.
Can you think of a better way of doing it?